### PR TITLE
upd ryu

### PR DESCRIPTION
**Pull request by RyujinZX**
1) Paimon Follow
2) Skip Cutscene (Movie)
3) Profile Changer (uid, level, img, card, nickname ...)
4) Fix AutoCooking

### CODE REVIEW

**Comment by Taiga74164** [comment]
image

**Comment by Taiga74164** [comment]
5. HideUI
6. In-game Browser

**Comment by Lunairayz** [comment]
WTF LMAOOO

**Comment by RyujinZX** [comment]
> WTF LMAOOO

More fan content ? =D

**Comment by tedzify** [comment]
Epic.
Screenshot (11)

File: cheat-library/src/user/cheat/visuals/Browser.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#include "pch-il2cpp.h"
+#include "Browser.h"
+
+#include <helpers.h>
+#include <cheat/events.h>
+#include <misc/cpp/imgui_stdlib.h>
+
+#include <cheat/esp/ESPRender.h>
```
Comment (Taiga74164) [inline cheat-library/src/user/cheat/visuals/Browser.cpp:8]: forgot to remove lmao

**Pull request merged**